python/usb_console/radio_session.py

```diff
@@ -56,158 +61,62 @@ def __init__(self, device_name, datastore, logger, radio_keys_config):
         self.mtmsn=-1
```

Comment:
I think we should fetch the IMEI number as a separate constructor argument, rather than from radio_keys_config. The argument should be populated with the value from standard config data, e.g. see `fc_only_native.json` for an example. The reason for this is that the IMEI number is declared in the standard config files in order to tie an IMEI number.